build-tools/installers/create-installers.targets

```diff
@@ -194,6 +194,19 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Bindings.targets" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Tasks.dll" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Tasks.pdb" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\cs\Xamarin.Android.Build.Tasks.resources.dll" />
```

Comment:
@pjcollins, @brendanzagaeski: Should we create a `@(_LocalizationLanguages)` item group and use an "expansion" here?

```xml
<ItemGroup>
  <_LocalizationLanguages Include="cs" />
  <_LocalizationLanguages Include="de" />
  <!-- ... -->
</ItemGroup>

<!-- ... -->

  <_MSBuildFiles Include="@(_LocalizationLanguages->'$(MSBuildSrcDir)\%(Identity)\Xamarin.Android.Build.Tasks.resources.dll')" />
```

Reply:
I think an expansion would be good here.